examples/cms/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const PLUGIN_KEYS = {
+    COLLECTION_ID: "collectionId",
+    SLUG_FIELD_ID: "slugFieldId",
+} as const
+
+export const UI_DEFAULTS = {
+    SETUP_WIDTH: 320,
+    SETUP_HEIGHT: 305,
+    MAPPING_WIDTH: 360,
+    MAPPING_HEIGHT: 425,
```

Comment:
Instead of SETUP and MAPPING I would use the same names as the components you use for rendering. Data source selection and field mapping. And now that all UI is controlled from the app component. Isn't it easier to just inline these values?